lua/lazyvim/plugins/extras/editor/overseer.lua

```diff
@@ -0,0 +1,103 @@
+return {
+  {
+    "catppuccin",
+    optional = true,
+    opts = {
+      integrations = { overseer = true },
+    },
+  },
+  {
+    "folke/which-key.nvim",
+    optional = true,
+    opts = {
+      defaults = {
+        ["<leader>o"] = { name = "+overseer" },
+      },
+    },
+  },
+  {
+    "stevearc/overseer.nvim",
+    cmd = {
+      "OverseerOpen",
+      "OverseerClose",
+      "OverseerToggle",
+      "OverseerSaveBundle",
+      "OverseerLoadBundle",
+      "OverseerDeleteBundle",
+      "OverseerRunCmd",
+      "OverseerRun",
+      "OverseerInfo",
+      "OverseerBuild",
+      "OverseerQuickAction",
+      "OverseerTaskAction",
+      "OverseerClearCache",
+    },
+    {
+      "folke/edgy.nvim",
+      optional = true,
+      opts = function(_, opts)
+        opts.right = opts.right or {}
+        table.insert(opts.right, {
+          title = "Overseer",
+          ft = "OverseerList",
+          open = function()
+            require("overseer").open()
+          end,
+        })
+      end,
+    },
+    opts = {
```

Comment:
this and keys shold be under the overseer.nvim plugin

Reply:
Already fixed.